pkg/kubelet/container/runtime.go

```diff
@@ -176,6 +176,8 @@ type RunContainerOptions struct {
 	// into docker's container runtime.
 	NetMode string
 	IpcMode string
+	// The parent cgroup to pass to Docker
+	CgroupRoot string
```

Comment:
I'm not blocking it.

On Fri, Apr 24, 2015 at 3:10 PM, Victor Marmol <notifications@github.com>
wrote:

> In pkg/kubelet/container/runtime.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7277#discussion_r29090057>
> :
>
> > @@ -176,6 +176,8 @@ type RunContainerOptions struct {
> >  	// into docker's container runtime.
> >  	NetMode string
> >  	IpcMode string
> > +	// The parent cgroup to pass to Docker
> > +	CgroupRoot string
>
> This is the only change that remains in case you missed it @guenter
> <https://github.com/guenter>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7277/files#r29090057>
> .
>